Mathlib/NumberTheory/Dioph.lean

```diff
@@ -343,14 +343,10 @@ theorem union : ∀ (_ : Dioph S) (_ : Dioph S'), Dioph (S ∪ S')
             (exists_congr fun t =>
               (@mul_eq_zero _ _ _ (p ((v ⊗ t) ∘ (inl ⊗ inr ∘ inl)))
                   (q ((v ⊗ t) ∘ (inl ⊗ inr ∘ inr)))).symm))
-      -- Porting note: putting everything on the same line fails
-      · refine inject_dummies_lem _ ?_ ?_ _ _
-        · exact some ⊗ fun _ => none
-        · exact fun _ => by simp only [elim_inl]
-      -- Porting note: putting everything on the same line fails
-      · refine inject_dummies_lem _ ?_ ?_ _ _
-        · exact (fun _ => none) ⊗ some
-        · exact fun _ => by simp only [elim_inr]⟩
+      · refine inject_dummies_lem _ (some ⊗ fun _ => none) ?_ _ _
+        exact fun _ => by simp only [elim_inl]
+      · refine inject_dummies_lem _ ((fun _ => none) ⊗ some) ?_ _ _
+        exact fun _ => by simp only [elim_inr]⟩
```

Comment:
Does it still fail if everything goes on the same line?

Reply:
Yes. Restored note.